### PR TITLE
Update DALI trademark information

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It updates DALI trademark information

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     Update the main readme
 - Affected modules and functionalities:
     README
 - Key points relevant for the review:
     NA
 - Validation and testing:
     NA
 - Documentation (including examples):
     Yes

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: README.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ pre-processing implementations, resulting in challenges such as
 portability of training and inference workflows, and code
 maintainability.
 
-DALI (R), the NVIDIA (R) Data Loading Library, is a collection of highly
+DALI NVIDIA (R) (Data Loading Library), is a collection of highly
```
Comment (jantonguirao) [inline README.rst:19]:
```suggestion
NVIDIA DALI (R) (Data Loading Library), is a collection of highly
```
Wasn't it NVIDIA DALI?

Comment (JanuszL) [reply README.rst]: Fixef

**Comment by JanuszL** [comment]
!build

**Review by banasraf — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ pre-processing implementations, resulting in challenges such as
 portability of training and inference workflows, and code
 maintainability.
 
-DALI (R), the NVIDIA (R) Data Loading Library, is a collection of highly
+NVIDIA(R) DALI(R) (Data Loading Library), is a collection of highly
```
Comment (klecki) [inline README.rst:19]:
```suggestion
NVIDIA(R) DALI(R), NVIDIA's Data Loading Library, is a collection of highly
```

Comment (JanuszL) [reply README.rst:19]: Done

**Comment by JanuszL** [comment]
!build

**Review by klecki — APPROVED** [review]

**Pull request merged**